IMPORTING-DATA.md

```diff
@@ -294,6 +294,23 @@ run it on rather than replaying traffic from production. Useful when
 upgrading production environments, perhaps less so for upgrading other
 environments.
 
+For a more comprehensive test, you can use the `test-samples.sh`
+script, which needs to be run before and after a database upgrade:
+
+    $ your laptop> scp test-samples.sh mapit-1.production
```

Comment:
This will already be on the server, right?

Reply:
Yeah, I guess it will be in `/var/apps/mapit` once this PR is merged